### PR TITLE
fix(lang): improve Italian labels

### PR DESCRIPTION
**Pull request by astagi**
## Description

This PR improves Italian labels using a more friendly and direct language

## Requirements Checklist
- [ ] Feature implemented / Bug fixed
- [ ] If necessary, more likely in a feature request than a bug fix
  - [ ] Change has been verified in an actual browser (Chrome, Firefox, IE)
  - [ ] Unit Tests updated or fixed
  - [ ] Docs/guides updated
  - [ ] Example created ([starter template on JSBin](https://codepen.io/gkatsev/pen/GwZegv?editors=1000#0))
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

File: lang/it.json [hunk 1]
```diff
@@ -71,14 +71,14 @@
   "Proportional Serif": "Carattere a spaziatura variabile con grazie (Proportional Serif)",
```
Comment (mister-ben) [inline lang/it.json:71]: Not related to your changes, but noting while I'm here that these are way too long for the captions settings box where they're displayed.

**Review by mister-ben — APPROVED** [review]
Grazie!

File: lang/it.json (hunk 1 above)
Comment (astagi) [reply lang/it.json:71]: Agree, these look to be literal translations :) I'll fix them in another commit using "Serif proporzionale" and "Serif monospazio" labels

File: lang/it.json [hunk 2]
```diff
@@ -66,19 +66,19 @@
   "Uniform": "Uniforme",
   "Dropshadow": "Ombreggiatura",
   "Font Family": "Famiglia di caratteri",
-  "Proportional Sans-Serif": "Carattere a spaziatura variabile senza grazie (Proportional Sans-Serif)",
-  "Monospace Sans-Serif": "Carattere a spaziatura fissa senza grazie (Monospace Sans-Serif)",
-  "Proportional Serif": "Carattere a spaziatura variabile con grazie (Proportional Serif)",
-  "Monospace Serif": "Carattere a spaziatura fissa con grazie (Monospace Serif)",
+  "Proportional Sans-Serif": Sans-Serif proporzionale",
```
Comment (mister-ben) [inline lang/it.json:69]:
```suggestion
  "Proportional Sans-Serif": "Sans-Serif proporzionale",
```
missing a quote here

**Review by mister-ben — CHANGES_REQUESTED** [review]

File: lang/it.json (hunk 2 above)
Comment (astagi) [reply lang/it.json:69]: Whoops! sorry, I'll amend my last commit

Comment (astagi) [reply lang/it.json]: Done @mister-ben

**Review by mister-ben — APPROVED** [review]

**Review by misteroneill — APPROVED** [review]

**Pull request merged**